geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/tables/AttributeTable.scala

```diff
@@ -48,36 +49,86 @@ object AttributeTable extends GeoMesaTable with LazyLogging {
 
   private type TryEncoder = Try[(TypeEncoder[Any, String], TypeEncoder[_, String])]
 
-  override def supports(sft: SimpleFeatureType) =
-    sft.getSchemaVersion > 5 && sft.getAttributeDescriptors.exists(_.isIndexed)
+  override def supports(sft: SimpleFeatureType) = {
+    val ok = sft.getSchemaVersion > 5 && sft.getAttributeDescriptors.exists(_.isIndexed)
+    if (ok && sft.getVisibilityLevel == VisibilityLevel.Attribute &&
+        !sft.getAttributeDescriptors.filter(_.isIndexed).forall(_.getIndexCoverage() == IndexCoverage.FULL)) {
+      // TODO GEOMESA-1254 support index values
+      throw new IllegalArgumentException("Attribute level visibility is currently only supported for fully" +
```

Comment:
I figure it's better to fail fast, rather than silently suppress indexing of attributes (which returning false would do)